index.html

```diff
@@ -13,12 +13,17 @@
     <title>mobile skeleton</title>
 </head>
 <body>
-    <section>
-    <div class="main-mobile">
+   
+    <section class="main-mobile">
     <header>
         <nav>
             <a href="#"><h2 id="welcome">Welcome</h2></a>
             <div class="nav-links">
+                <ul>
+                    <li><a href="Portfolio">Portfolio</a></li>
+                    <li><a href="About">About</a></li>
+                    <li><a href="Contact">Contact</a></li>
+                </ul> 
```

Comment:
- [ ] Kindly be noticed that the menu items in the `nav` bar is a little bit to left, please fix that and make looks like the Figma design.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-08-25 16-41-57](https://user-images.githubusercontent.com/33752735/186696780-6d99340c-4ce1-4f4a-ac7c-11fdc8ae235e.png) | ![Screenshot from 2022-08-25 16-48-21](https://user-images.githubusercontent.com/33752735/186697199-da13075a-0137-4271-8150-f9ba7b110d73.png)